melos.yaml

```diff
@@ -20,8 +20,10 @@ scripts:
     description: Run all static analysis checks.
 
   analyze:
+    # We are setting the concurrency to 1 because a higher concurrency can crash
+    # the analysis server on low performance machines (like GitHub Actions).
     run: |
-      melos exec -c 5 -- \
+      melos exec -c 1 -- \
```

Comment:
This fixes the `The analysis server shut down unexpectedly.` issue.

Reply:
I will also create a ticket in the Dart repo in the next days 👍

Reply:
Ok, to be fair enough, sometimes the analysis server still crashes: https://github.com/nilsreichardt/flutterfire/runs/6899981134?check_suite_focus=true. But it is <0.5%

Reply:
> I will also create a ticket in the Dart repo in the next days 👍

https://github.com/dart-lang/sdk/issues/49272